application/src/utils/validateBasket.ts

```diff
@@ -0,0 +1,32 @@
+import { FormikErrors } from 'formik';
+
+type FormValues = {
+  email: string;
+  surname: string;
+  name: string;
+  mobile: string;
+};
+const VALIDATE_EMAIL = new RegExp(
+  '^[A-Z0-9._%+-]+@[A-Z0-9.-]+.[A-Z]{2,}$',
+  'i'
+);
```

Comment:
Let's create `utils` folder and create functions which will be responsible for email validations

```
const isEmailValid = (email) => {
... your code
}
```

Reply:
done